content/docs/components-and-props.md

```diff
@@ -122,13 +121,13 @@ ReactDOM.render(
 
 [](codepen://components-and-props/composing-components)
 
-Typically, new React apps have a single `App` component at the very top. However, if you integrate React into an existing app, you might start bottom-up with a small component like `Button` and gradually work your way to the top of the view hierarchy.
+Por lo general, aplicaciones nuevas en React tienen un único componente `App` en lo mas alto. Sin embargo, si se integra React en una aplicación existente, se podría empezar de abajo hacia arriba con un pequeño componente como `Button` y poco a poco trabajar el camino a la cima de la jerarquía de la vista.
 
-## Extracting Components
+## Extracción de Componentes
 
-Don't be afraid to split components into smaller components.
+No tengas miedo de dividir los componentes en unos más pequeños.
```

Comment:
While I get why you used `unos`, in this case I think it's important to be as close as possible to the original text.

So my suggestion would be:
> No tengas miedo de dividir los componentes en componentes más pequeños.

Which I believe sounds better.

Reply:
What about `otros mas pequeños` ?

Reply:
> What about `otros mas pequeños` ?

Yeah that's cool for me.